lib/Dialect/FIRRTL/Transforms/ResolveTraces.cpp

```diff
@@ -148,14 +148,21 @@ struct ResolveTracesPass : public ResolveTracesBase<ResolveTracesPass> {
 
     // If this is operation doesn't have a single result (and no way to know
     // what its type is) or if it doesn't have a name, then do nothing.
```

Comment:
Maybe this comment should move down with the `getNumResults()` check?

Reply:
Updated this comment block, thanks!